packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -14,7 +14,7 @@
 
 @mixin card {
   .#{$prefix}--card {
-    color: $text-01;
```

Comment:
By removing this color you've fixed #1404 as well it seems like. (although I'm not a fan of how the blue is coming from `bx--link` instead of our `scss`).

This introduces a new problem within the locale selector...

<img src="https://user-images.githubusercontent.com/3793636/74494149-d5abf280-4e99-11ea-858c-64604a71bb2d.png" alt="" width="400px" />

The icons are supposed to be `$text-01` (this card was originally built for the local selector which is why it was `$text-01` to begin with.)

Reply:
the blue is explicitly defined now like so
```
&__cta,
&__cta a:visited {
      color: $interactive-04;
}
```